Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -6799,7 +6812,7 @@ private ElementInfo GetElementInfoFromIWebElement(IWebElement el, HtmlNode htmlN
             {
                 EI.XPath = ChildElementInfo.XPath.Substring(0, ChildElementInfo.XPath.LastIndexOf("/"));
             }
-            EI.ElementObject = el;
+            EI.ElementObject = webElement;              // el;
```

Comment:
do we still need GetElementInfoFromIWebElement()? and GetHTMLElementInfoFromIWebElement()?   can we replace both with LearnElementInfoDetails()?